src/librustdoc/clean/types.rs

```diff
@@ -1407,34 +1407,34 @@ pub(crate) struct Function {
 
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 pub(crate) struct FnDecl {
-    pub(crate) inputs: Arguments,
+    pub(crate) inputs: Params,
     pub(crate) output: Type,
     pub(crate) c_variadic: bool,
 }
 
 impl FnDecl {
     pub(crate) fn receiver_type(&self) -> Option<&Type> {
-        self.inputs.values.first().and_then(|v| v.to_receiver())
+        self.inputs.0.first().and_then(|v| v.to_receiver())
     }
 }
 
+/// The parameter list of a function.
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
-pub(crate) struct Arguments {
-    pub(crate) values: Vec<Argument>,
-}
+pub(crate) struct Params(pub(crate) Vec<Param>);
```

Comment:
Since there is no method implemented on this type, is it useful to create a new type for it? I precise: I don't mind having a type, just curious about your logic.

Also, please use full name: `Parameters`.

Reply:
I think it's a good idea. :+1: